java/Cherepnin_Oleh/src/com/zoolatech/lecture6/tasks/_2/Order.java

```diff
@@ -0,0 +1,56 @@
+package com.zoolatech.lecture6.tasks._2;
+
+import java.util.Objects;
+
+public class Order {
+    private String id;
+    private StoreType type;
+    private float price;
+    private String country;
+    private static int idCounter = 1;
+
+    public Order(StoreType type, float price, String country) {
+        this.id = String.valueOf(idCounter);
+        this.type = type;
+        this.price = price;
+        this.country = country;
+        idCounter++;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
same as in the previous task: equals, hashCode and toString should be defined at the end of the class, instance fields can be accessed directly